nox/virtualenv.py

```diff
@@ -52,12 +52,15 @@ def __init__(self, bin=None, env=None):
     def bin(self):
         return self._bin
 
-    def run(self, args, in_venv=True):
+    def run(self, args, in_venv=True, env=None):
         """Runs a command. By default, the command runs within the
         environment."""
+        if in_venv:
+            env = env or {}
+            env.update(self.env)
```

Comment:
We should be careful when modifying a parameter that was passed in. I'd prefer if we did `env.copy` here.